borg/archiver.py

```diff
@@ -731,6 +741,10 @@ def format_cmdline(cmdline):
         stats = archive.calc_stats(cache)
         print('Name:', archive.name)
         print('Fingerprint: %s' % hexlify(archive.id).decode('ascii'))
+
+        if archive.metadata.get(b'comment') is not None:
+            print('Comment:', archive.metadata[b'comment'])
```

Comment:
The current implementation has the issue that while the comment can be modified it cannot be removed (set to None). `''` as the default would fix that. However, I think it's better to not show the `Comment:` line when the comment is not set (or is empty in this case).